internal/database/gitserver_repos.go

```diff
@@ -165,12 +165,12 @@ type IteratePurgableReposOptions struct {
 	// DeletedBefore will filter the deleted repos to only those that were deleted
 	// before the given time. The zero value will not apply filtering.
 	DeletedBefore time.Time
-	// Limit optionally limtits the repos iterated over. The zero value means no
-	// limits are applied. Repos are ordered by their deleted at date, oldest first.
+	// Limit optionally limits the repos iterated over. The zero value means no
+	// limits are applied. Repos are ordered by their deleted at date, the oldest first.
```

Comment:
Nit: Just `oldest first` sounds right to me or `ordered by oldest first`.

```suggestion
	// limits are applied. Repos are ordered by their deleted at date, ordered by oldest first.
```